.eslintrc.js

```diff
@@ -73,6 +73,14 @@ module.exports = {
     'valid-jsdoc': 0,
     'arrow-parens': 0,
   },
+  overrides: [
+    {
+      files: ['*expectations.js'],
```

Comment:
maybe narrow this more? e.g. cli/test/smokehouse

Reply:
done